client/service-worker.js

```diff
@@ -38,12 +38,13 @@ self.addEventListener("fetch", function (event) {
 
 	const path = url.substring(scope.length);
 
-	// Skip ignored paths
-	if (excludedPathsFromCache.test(path)) {
+	if (!includedPathsInCache.test(path)) {
 		return;
 	}
 
-	event.respondWith(networkOrCache(event));
+	const response = networkOrCache(event);
```

Comment:
Why did I split this up?

Reply:
Because there were like a million conditionals in there :stuck_out_tongue_winking_eye: 
You've eliminated ~~all~~ most of them since then